.github/workflows/daily-build.yml

```diff
@@ -17,12 +17,12 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: 17.0.7
-
+      
       - name: Set environment variable
```

Comment:
This is workflow is not centralized. Is this because this module has compiler plugin?

Reply:
https://github.com/ballerina-platform/module-ballerinax-kafka/blob/master/.github/workflows/daily-build.yml